api/MediaSource.json

```diff
@@ -456,26 +463,25 @@
           }
         }
       },
-      "onsourceclose": {
+      "readyState": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaSource/onsourceclose",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaSource/readyState",
+          "spec_url": "https://w3c.github.io/media-source/#dom-mediasource-readystate",
           "support": {
             "chrome": {
-              "version_added": "53"
+              "version_added": "23"
             },
             "chrome_android": {
               "version_added": "33"
             },
             "edge": {
-              "version_added": "17"
+              "version_added": "12"
```

Comment:
Can you research the Edge version numbers or revert and file an issue for follow-up? Probably it was just missing event handler properties in Edge too...

Reply:
Added ranges for Edge. Assumed initial support in 12 and on-eventhandlers in 17.

The on-eventhandlers appear here: https://docs.microsoft.com/en-us/archive/microsoft-edge/legacy/developer/dev-guide/whats-new/edgehtml-17
The main parts of the API were added in 12: https://docs.microsoft.com/en-us/archive/microsoft-edge/legacy/developer/dev-guide/whats-new/edgehtml-12

Reply:
Thanks! That's probably correct then, I won't try to verify.